src/main/java/bobthebot/tasks/Task.java

```diff
@@ -0,0 +1,60 @@
+package bobthebot.tasks;
+
+/**
+ * Class representing tasks.
+ * */

```

Comment:
```suggestion
 *
 */
```
[relevant coding standard - subsequent * should be aligned with the first one](https://se-education.org/guides/conventions/java/intermediate.html#:~:text=Subsequent%20*%20is%20aligned%20with%20the%20first%20one)